src/features/tags/selectors.ts

```diff
@@ -0,0 +1,79 @@
+import { NAME } from "./constants";
+import { Tag } from "./types";
+
+// does created_at get updated whenever a row is updated in mysql?
+export const getMostRecentTags = (
+  state: any,
+  list: string
+): Tag[] | undefined => {
+  // give time for tags to load
+  if (state.tagsByList[`${list}`] && state.tagsByList[`${list}`].items) {
+    let tagsArray = state.tagsByList[`${list}`].items.map(
+      (id: number) => state[NAME][id]
+    );
+
+    let tags = tagsArray
+      .sort((a: Tag, b: Tag) => {
+        if (a.date > b.date) return 1;
+        else return -1;
+      })
+      .map((tag: Tag) => state[NAME][tag.id].id)
+      .reverse()
+      .slice(0, 10);
+
+    return tags;
+  }
+};
+
+export const getTopTags = (state: any, list: string): Tag[] | undefined => {
```

Comment:
Same here. This is returning the top tag ids. So the return type is `number[] | undefined`